### ISSUE TITLE
Wrong include path is added to compile command line

### ISSUE THREAD

**Issue by Nicoretti** [esrlabs/bake#67]
Assuming the following project structure
```
PROJECT_NAME
|-- Project.meta
|-- PROJECT_NAME
    |-- include
    `-- src
```
if in the Project.meta PROJECT_NAME/include is specified as include name only `-Iinclude` will be added to the compile commandline, but it should be `-IPROJECT_NAME/include`.

**Comment by aschaal** [comment]
This is intended and documented. However, it is not obvious if you encounter this behaviour the first time. -> I will add a FAQ section and print a info message if there is local path which would also match.

**Comment by aschaal** [comment]
Added info output. Should be enough. FAQ may be introduced when docu will be refactored...

**Comment by aschaal** [comment]
Fixed in 2.15.0.
